File: DAY 5/JAVA/AtmCard.java

import java.util.Scanner;
public class AtmCard{
public static void main(String... arg){


	Scanner input = new Scanner(System.in);

	System.out.printf("Enter atm card number :" );
	String atmCardNumber = input.nextLine();

	/**
	atmCardNumber.index(4) = ("-");
	atmCardNumber.index(9) = ("-");
	atmCardNumber.index(14) = ("-");
	**/


	System.out.println(cardValidator(atmCardNumber));



}

	public static boolean cardValidator(String atmCardNumber){

		
		int cardlength = atmCardNumber.length();

		char firstindex = atmCardNumber.charAt(0);
		
		char forthindex = atmCardNumber.charAt(4);
		
		char ninethindex = atmCardNumber.charAt(9);

		char fourteentindex = atmCardNumber.charAt(14);



		if ((firstindex == ('4') || firstindex == ('5') || firstindex == ('6')) && (cardlength == 19) && (forthindex == '-' || ninethindex == '-' || fourteentindex == '-')){
		return true;


		}else return false;

		


	}

  



}

File: DAY 5/PYTHON/AtmCard.py
#AtmCard

cardNumber = str(input('input:'  ));

cardLength = len(cardNumber)


index0 = cardNumber[0]
index4 = cardNumber[4]
index9 = cardNumber[9]
index14 = cardNumber[14]



print(index0)
print(index4)
print(index9)
print(index14)


if (index0 == '4' or index0 == '5' or index0 == '6' and cardLength == 19):
	print('Valid card')


elif (cardLength != 19):

	print('wrong card length') 

else: print('invalid Card')

#print(cardNumber)


"""
print(index0)
print(index4)
print(index9)
print(index14)
"""